serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-custom-dashboard-page.adoc

```diff
@@ -0,0 +1,23 @@
+= Dashboards in Kogito Serverless Workflow Tools extension
```

Comment:
The page is missing those:
```
:compat-mode!:
// Metadata:
:description: ...
:keywords: kogito, workflow, serverless, Quarkus, Dev UI, dashboards
```

Reply:
See https://raw.githubusercontent.com/kiegroup/kogito-docs/main/serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc as an example